infra/modules/s3/pdf-processing-lambda-src/index.js

```diff
@@ -209,15 +210,16 @@ const mapPublicationVersionToMetadata = (publicationVersion, pdfUrl) => {
       },
       article: {
         title: publicationVersion.title,
-        type: publicationVersion.publication.type,
+        type: publication.type,
         version: "VOR",
         language: [publicationVersion.language],
         identifier: [
           {
             type: "doi",
-            id: publicationVersion.publication.doi,
+            id: publication.doi,
           },
         ],
+        e_num: publication.id, // DOI suffix e.g. "abcd-efgh"
```

Comment:
Curious why e_num and not just enum? Is it a reserved word?

Reply:
This is part of the publications router spec so not something we've chosen - it's short for "Electronic article number" (it may be a reserved word somewhere too)